src/NearOrg/Notifications/Notification.jsx

```diff
@@ -0,0 +1,99 @@
+const accountId = props.accountId;
+const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
+const profileUrl = `#/near/widget/ProfilePage?accountId=${accountId}`;
+
+const username = props.username;
+const action = props.action;
+const componentName = props.componentName;
+const timestamp = props.timestamp;
+const desc = props.desc;
+
+const Notification = styled.div`
+  display: flex;
+  width: 470px;
+  padding: 16px 24px 16px 16px;
+  align-items: flex-start;
+  gap: 16px;
+  border-top: 1px solid var(--sand-light-6, #e3e3e0);
+
+  &:hover {
+    background: var(--sand-light-2, #f9f9f8);
+  }
+`;
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+  flex: 1 0 0;
+`;
+
+const Icon = styled.div``;
+
+const Username = styled.span`
+  font: var(--text-s);
+  font-weight: 600;
+`;
+
+const Action = styled.span`
+  font: var(--text-s);
+  color: #706f6c;
+`;
+
+const ComponentName = styled.span`
+  font: var(--text-s);
+  font-weight: 600;
+  color: #604cc8;
+`;
+
+const Timestamp = styled.span`
+  font: var(--text-s);
+  color: #706f6c;
+`;
+
+const Desc = styled.span`
+  font: var(--text-s);
+  color: #706f6c;
+  font-style: italic;
+  border-left: 2px solid #e3e3e0;
+  padding: 0 0 0 1rem;
+`;
+
+return (
+  <>
+    <Notification>
+      <Icon>
+        <svg
+          width="20"
+          height="20"
+          viewBox="0 0 20 20"
```

Comment:
You can simply use this icon instead:

```suggestion
      <i class="ph ph-git-fork" />
```

I check it here https://phosphoricons.com/ and it looks exact the same